cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/TransactionHttpHandler.java

```diff
@@ -175,4 +208,78 @@ public void resetTxManagerState(HttpRequest request, HttpResponder responder) {
     txClient.resetState();
     responder.sendStatus(HttpResponseStatus.OK);
   }
+
+  @Path("/transactions/pruneinfo")
```

Comment:
Should we make this `/transactions/prune-info` for readability?
@awholegunch What do you suggest?